recipes/docker_registry.rb

```diff
@@ -102,47 +102,52 @@
 base_filename = File.basename(image_url)
 download_command = " wget #{image_url}"
 
-if node['install']['enterprise']['install'].casecmp? "true"
+base_image = "#{registry_host}:#{node['hops']['docker']['registry']['port']}/#{node['hops']['docker']['base']['image']['name']}:#{node['install']['version']}"
+base_image_python = "#{registry_host}:#{node['hops']['docker']['registry']['port']}/#{node['hops']['docker']['base']['image']['python']['name']}:#{node['install']['version']}"
+
+if node['install']['enterprise']['install'].casecmp? "fixme"
```

Comment:
This is something I added temporarily so I could override the url for the docker images. Once we merge the docker-images PR it will upload the .tar here.